ui/shared/components/panel/variants/Frequencies.jsx

```diff
@@ -141,9 +141,47 @@ gnomadLink.propTypes = {
   fieldTitle: PropTypes.string,
 }
 
+const GNOMAD_URL_INFO = {
+  urls: { [GENOME_VERSION_38]: 'gnomad.broadinstitute.org' },
+  queryParams: { [GENOME_VERSION_38]: 'dataset=gnomad_r3' },
+}
+
+const sectionTitle = ({ fieldTitle, section }) => (
+  <span>
+    {fieldTitle}
+    &nbsp;
+    {section.toLowerCase()}
+  </span>
+)
+
+sectionTitle.propTypes = {
+  fieldTitle: PropTypes.string,
+  section: PropTypes.string,
+}
+
+const HOM_SECTION = 'Homoplasmy'
+const HET_SECTION = 'Heteroplasmy'
+const AC_SECTION = 'Allele Counts'
+const GLABLE_AF_SECTION = 'Global AFs'
+const SECTIONS = [GLABLE_AF_SECTION, AC_SECTION, HOM_SECTION, HET_SECTION]
+
 const POPULATIONS = [
   { field: 'sv_callset', fieldTitle: 'This Callset', acDisplay: 'AC', helpMessage: SV_CALLSET_CRITERIA_MESSAGE },
-  { field: 'callset', fieldTitle: 'This Callset', acDisplay: 'AC' },
+  {
+    field: 'callset',
+    fieldTitle: 'This Callset',
+    acDisplay: 'AC',
+    titleContainer: sectionTitle,
```

Comment:
this will break the display for non-mito variants

Reply:
Oh, yes. It used to have an `isMito` parameter for the `sectionTitle` function. I removed the parameter at some point.